src/org/traccar/notification/NotificationForward.java

```diff
@@ -0,0 +1,71 @@
+package org.traccar.notification;
+
+import java.nio.charset.StandardCharsets;
+
+import javax.json.Json;
+import javax.json.JsonObjectBuilder;
+
+import org.traccar.Context;
+import org.traccar.model.Device;
+import org.traccar.model.Event;
+import org.traccar.model.Geofence;
+import org.traccar.model.Position;
+import org.traccar.web.JsonConverter;
+
+import com.ning.http.client.AsyncHttpClient.BoundRequestBuilder;
+
+public final class NotificationForward {
+
+    private NotificationForward() {
+    }
+
+    private static final String USER_AGENT = "Traccar Server";
+
+    private static final String KEY_POSITION = "position";
+    private static final String KEY_EVENT = "event";
+    private static final String KEY_GEOFENCE = "geofence";
+    private static final String KEY_DEVICE = "device";
+
+    public static void forwardEvent(Event event, Position position) {
```

Comment:
I think it's better to add this class to `Context` instead of having static method. It would be more consistent with other modules.